lib/matplotlib/axes/_axes.py

```diff
@@ -5787,11 +5787,14 @@ def imshow(self, X, cmap=None, norm=None, *, aspect=None,
             which can be set by *filterrad*. Additionally, the antigrain image
             resize filter is controlled by the parameter *filternorm*.
 
-        interpolation_stage : {'data', 'rgba'}, default: 'data'
-            If 'data', interpolation
-            is carried out on the data provided by the user.  If 'rgba', the
-            interpolation is carried out after the colormapping has been
-            applied (visual interpolation).
+        interpolation_stage : {'antialiased', 'data', 'rgba'}, default: 'antialiased'
+            If 'data', interpolation is carried out on the data provided by the user.
+            If 'rgba', the interpolation is carried out in RGBA-space after the
```

Comment:
Optionally, give a hint when these are suited:

'data' is well suited for creating intermediate values when up-scaling. 'rgba' reduces anti-aliasing when down-scaling.

Reply:
Does the next sentence explaining the default basically gives the hint?

Reply:
Yes, it gives a hint, but only a hint. There could be more and clearer information.

It says which is used in which case. One may assume that these are the best respective options for the cases, but it's not explicitly stated (There could be other conditions involved in the decision for 'antialiased'). Also not stated is the respective benefit "suited for creating intermediate values" / "reduce anti-aliasing".

If we only had "antialiased", that would be ok, because it essentially communicates "you get some nice rendering". But if you have the other options, you may want to know what they do exactly, when you want to choose them and what the advantage/disadadvange is.

Reply:
OK, Changed to 
```
            If 'data', interpolation is carried out on the data provided by the user,
            useful if interpolating between pixels during upsampling.
            If 'rgba', the interpolation is carried out in RGBA-space after the
            color-mapping has been applied, useful if downsampling and combining
            pixels visually. The default 'antialiased' is appropriate for most
            applications where 'rgba' is used when downsampling, or upsampling at a
            rate less than 3, and 'data' is used when upsampling at a higher rate.
            See :doc:`/gallery/images_contours_and_fields/image_antialiasing` for
            a discussion of image antialiasing.

```